example/module.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  localModules: [
+    { moduleName: '@edx/paragon/scss/core', dir: '..', dist: 'scss/core' },
+    { moduleName: '@edx/paragon/icons', dir: '..', dist: 'icons' },
+    { moduleName: '@edx/paragon', dir: '..', dist: 'dist' },
```

Comment:
We could probably use `dist: 'src'` here so the example app gets hot reloading of file changes in the `src` directory without requiring a full `npm run build` to get files in `dist`?